static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -133,6 +134,12 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
         })
       : false;
 
+  const isMobile =
+    isMobilePlatform(event.platform) ||
+    (event.platform === 'other' &&
+      isMobilePlatform(event.release?.projects[0].platform)) ||
+    (event.platform === 'java' && isMobilePlatform(event.release?.projects[0].platform));
```

Comment:
since we're using this same logic twice can we add a function directly inside stacktraceLink that takes an event and decides if its a supported mobile platform.

I think this would be easy to not update in both places if we make changes in the future

Reply:
Thank you, I have implemented your feedback